edx_django_utils/monitoring/internal/backends.py

```diff
@@ -50,6 +50,13 @@ def record_exception(self):
         Record the exception that is currently being handled.
         """
 
+    @abstractmethod
+    def trace(self, name):
+        """
+        Create a manual trace with the given name. The underlying implementation is generally
+        assumed to be a contextmanager, and the level above calls it as such.
```

Comment:
This might be clearer, see what you think:

```suggestion
        Start a tracing span with the given name, returning a context manager instance.
        
        The caller must use the return value in a `with` statement or similar so that the
        span is guaranteed to be closed appropriately.
        
        Implementations should create a new child span parented to the current span,
        or create a new root span if not currently in a span.
```